src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -130,38 +141,90 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
     clusterCreateResponse shouldEqual clusterGetResponse
   }
 
+  it should "create and get a cluster via v2 API" in isolatedDbTest {
+    implicit val patienceConfig = PatienceConfig(timeout = 1.second)
```

Comment:
For some reason I had to specify the `timeout` for this one as the default one wasn't long enough...perhaps because there are more checks than usual?  🤔